### PR TITLE
308 sign up notif

### LINKED ISSUE #308: [FRONTEND] Create component telling unpaid users to sign up
**Is your feature request related to a problem? Please describe.**

Image

Should add appropriate links in the message to redirect users to the right page

**BEFORE MERGING**

- [ ] Storybooks created where possible
- [ ] Tested with [nock](https://tanstack.com/query/v3/docs/framework/react/guides/testing) if data fetching
- [ ] Tests written for critical interactions
- [ ] PR Reviewed (For non-trivial changes)
- [ ] Changes tested after rebasing on master or merging in master (_hint_: `git fetch origin master:master`, then `git rebase master` or `git merge master`)
- [ ] All required PR checks passing

### PR DESCRIPTION
**Pull request by zlrkw11**
closes #308

### CODE REVIEW

**Comment by zlrkw11** [comment]
image

File: client/src/components/generic/SignUpNotif/SignUpNotif.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export const SignUpNotif = () => {
+  const navigate = useNavigate()
+  function nav() {
+    navigate("/register")
+  }
+
+  return (
+    <div className="text-dark-blue-100 flex flex-col items-center justify-center gap-2">
+      <h2 className="text-center italic">Currently unavailable</h2>
+      <h4 className="h-[87px] w-[386px] text-center">
```
Comment (bcho892) [inline client/src/components/generic/SignUpNotif/SignUpNotif.tsx:13]: can we avoid hardcoding heights/widths

**Review by bcho892 — CHANGES_REQUESTED** [review]

File: client/src/components/generic/SignUpNotif/SignUpNotif.tsx (hunk 1 above)
Comment (zlrkw11) [reply client/src/components/generic/SignUpNotif/SignUpNotif.tsx]: done

Comment (zlrkw11) [reply client/src/components/generic/SignUpNotif/SignUpNotif.tsx]: done

**Review by bcho892 — APPROVED** [review]

**Pull request merged**